content/influxdb/cloud/account-management/limits.md

```diff
@@ -0,0 +1,114 @@
+{{% /warn %}}
+
+Review adjustable service quotas and global limits to plan for your bandwidth needs:
+
+- [Adjustable service quotas](#adjustable-service-quotas)
+- [Global limits](#global-limits)
+- [UI Error Messages](#ui-error-messages)
+<!-- - [API Error Responses](#api-error-responses) -->
+
+## Adjustable service quotas
+
+To reduce the chance of unexpected charges and protect the service for all users, InfluxDB Cloud has adjustable service quotas applied per account.
+
+_To request higher service quotas, reach out to [InfluxData Support](https://support.influxdata.com/)._
+
+### Free Plan
+
+- **Data-in**: Rate of 5 MB per 5 minutes (average of 17 kb/s)
+  - Uncompressed bytes of normalized [line protocol](/influxdb/cloud/reference/syntax/line-protocol/)
+- **Read**: Rate of 300 MB per 5 minutes (average of 1000 kb/s)
+  - Bytes in HTTP in response payload
+- **Cardinality**: 10k series (see [cardinality](/influxdb/cloud/reference/glossary/#series-cardinality))
+- **Available resources**:
+  - 2 buckets (excluding `_monitoring` and `_tasks` buckets)
+  - 5 dashboards
+  - 5 tasks
+- **Alerts**:
+  - 2 checks
+  - 2 notification rules
+  - Unlimited Slack notification endpoints
+- **Storage**: 30 days of data retention (see [retention period](/influxdb/cloud/reference/glossary/#retention-period))
+
+{{% note %}}
+Set your retention period to unlimited or up to 1 year by [updating a bucket’s retention period in the InfluxDB UI](/influxdb/cloud/organizations/buckets/update-bucket/#update-a-buckets-retention-period-in-the-influxdb-ui), or [set a custom retention period](/influxdb/cloud/organizations/buckets/update-bucket/#update-a-buckets-retention-period) using the [`influx` CLI](influxdb/cloud/reference/cli/influx/).
+{{% /note %}}
+
+{{% note %}}
+To write historical data older than 30 days, retain data for more than 30 days, or increase rate limits, upgrade to the Cloud [Usage-Based Plan](/influxdb/cloud/account-management/pricing-plans/#usage-based-plan).
+{{% /note %}}
+
+### Usage-Based Plan
+
+- **Data-in**: Rate of 3 GB per 5 minutes
+  - Uncompressed bytes of normalized [line protocol](/influxdb/cloud/reference/syntax/line-protocol/)
+- **Read**: Rate of 3 GB data per 5 minutes
+  - Bytes in HTTP in response payload
+- **Cardinality**: 1M series (see [cardinality](/influxdb/cloud/reference/glossary/#series-cardinality))
+- **Unlimited resources**
+  - dashboards
+  - tasks
+  - buckets
+  - users
+- **Alerts**
+  - Unlimited checks
+  - Unlimited notification rules
+  - Unlimited notification endpoints for [all endpoints](/flux/v0.x/tags/notification-endpoints/)
+
+## Global limits
+
+InfluxDB Cloud global system limits cannot be adjusted and apply to all accounts.
+These hard limits are typically dictated by the capabilities of the underlying InfluxDB Cloud infrastructure.
+Limits include:
+
+- Write request limits:
+  - 50 MB maximum HTTP request batch size (compressed or uncompressed--defined in the `Content-Type` header)
```

Comment:
I believe this is moving to 5 MB based on https://github.com/influxdata/k8s-idpe/pull/5886